sys-power/apcupsd/apcupsd-3.14.14-r4.ebuild

```diff
@@ -53,6 +57,10 @@ src_prepare() {
 }
 
 src_configure() {
+	# -Werror=odr
+	# https://bugs.gentoo.org/941584
+	filter-lto
+
 	local myconf
 
 	use cgi && myconf="${myconf} --enable-cgi --with-cgi-bin=/usr/libexec/${PN}/cgi-bin"
```

Comment:
If you're going to arrayise `myeconfargs` (which is fine/good), convert `myconf` too (switch all uses and make everything using it append instead)?

Reply:
Yeah, fine for me. Will do it :-) But not anymore today.

Reply:
This LTO issue was kind of trivial so I've attached a proper fix to the bug.

Reply:
Thank you very much for your patch. I have included it.